### DISCUSSION TITLE
Increase the visibility of menu flash

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#24949] [Ideas] [answered]
I'm checking different with menu flash between lazer and stable,
and I found https://github.com/ppy/osu/commit/0dd52e4e2930509788aa71a1eaaa47b2082a03d1
This commit attempts to reduce the effect of menu flash

But this commit is too old and there is no information. I don’t know what the purpose is.
Is it possible to modify it to have the same high visibility as stable?

pr is https://github.com/ppy/osu/pull/812 incredibly early

and I have a branch which try to increase the visibility of menu flash

Basically refer to the menu flash file in stable
[the menu flash file from stable (1x)](https://github.com/ppy/osu/assets/34775378/fc41510d-45cc-4b9f-b1c9-e3fdb385400c)

| before this pr | after this pr |
| :-: | :-: |
| ![image](https://github.com/ppy/osu/assets/34775378/9db7ddb8-806d-4db1-8747-4e9e167b1e00) |  ![image](https://github.com/ppy/osu/assets/34775378/5c8f0533-addd-4158-b093-2ac7ce69ab4e) |

<details>

<summary>video</summary>

https://github.com/ppy/osu/assets/34775378/9b17c80d-70b9-4bf0-9aac-20d343458532

https://github.com/ppy/osu/assets/34775378/46e63139-e8b7-400c-bd2a-153f2b8ea35e

</details>

**Comment by peppy** [comment, marked as answer]
It's intentional.
